airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -315,7 +315,7 @@ def field_to_property_schema(field_params: Mapping[str, Any]) -> Mapping[str, An
         elif sf_type == "boolean":
             property_schema["type"] = ["boolean", "null"]
         elif sf_type in LOOSE_TYPES:
-            property_schema["type"] = ["array", "boolean", "integer", "number", "object", "string", "null"]
+            property_schema["type"] = ["string", "integer", "number", "boolean", "array", "object", "null"]
```

Comment:
mb use set to not depend on order?

```
ANY_TYPES_SET = {"string", "integer", "number", "boolean", "array", "object", "null"}
...
set(property_schema["type"])  = ANY_TYPES_SET
```

Reply:
The overwhelming majority of data for the `anyType` type is `string` (more than 99%), and now there is such a problem with BigQuery that we transfer a schema with all possible types, while Salesforce sypports, and BigQuery takes only the first type (it was an array) and creates a field with this data type. And then we cannot put a string or other data type there, since we get the following [error](https://github.com/airbytehq/airbyte/issues/6886#issuecomment-963921760).